.idea/strimzi-mqtt-bridge.iml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```

Comment:
This file should be removed as well?

Reply:
Regarding [1], I will look into it further. But do we need the building in this PR?

Reply:
`.iml` is IntelliJ specific we don't want this stuff to be pushed. People can use any other IDE (i.e. VSCode) for developing.